solc/CommandLineParser.cpp

```diff
@@ -731,6 +732,12 @@ General Information)").c_str(),
 			po::value<string>()->value_name("all,bmc,chc,none")->default_value("none"),
 			"Select model checker engine."
 		)
+		(
+			g_strModelCheckerExtCalls.c_str(),
+			po::value<string>()->value_name("untrusted,trusted")->default_value("untrusted"),
+			"Select whether external calls should be considered untrusted or trusted"
+			" if the called function's code is available."
```

Comment:
Is this clear enough?
``Select whether to assume that external calls always invoke the code given by the type of the contract, if that code is available.`` or sth?